Minigames/src/main/java/au/com/mineauz/minigames/commands/HelpCommand.java

```diff
@@ -57,11 +57,11 @@ public boolean onCommand(CommandSender sender, Minigame minigame,
         if (sender instanceof Player) {
             player = (Player) sender;
         }
-        sender.sendMessage(ChatColor.GREEN + "List of Minigame commands");
+        sender.sendMessage(ChatColor.GREEN + MessageManager.getUnformattedMessage(null, "command.info.header"));
         sender.sendMessage(ChatColor.BLUE + "/minigame");
-        sender.sendMessage(ChatColor.GRAY + MessageManager.getMessage(null, "command.info.header"));
+        sender.sendMessage(ChatColor.GRAY + MessageManager.getUnformattedMessage(null, "command.info.mgm"));
         if (player == null || player.hasPermission("minigame.join")) {
-            sender.sendMessage(ChatColor.BLUE + MessageManager.getMessage(null, "command.info.mgm"));
+            sender.sendMessage(ChatColor.BLUE + "/minigame join <Minigame>");
```

Comment:
This seems to be reverting the use of `MessageManager` to a literal string, which we want to move away from.. was this a mistake?

Reply:
Oh also, these are exactly the same changes from #347 

Do you want to just remove these changes from this PR and we can deal with them separately in the other PR?

Reply:
Nope, the `command.info.mgm` was a different thing, not even related to the join command hahah. So realistically it was just the wrong message.
Also, all of the commands in this help command use this format for the usage. What I _could_ do is convert all of the usages to new message references though.

Reply:
Oh and you’re right. I accidentally pushed that commit to the same branch way back when I made the PRs… so it’s literally just a duplicate. It may be easier to just close this PR and merge the other one 🤷🏻‍♂️